src/Main.java

```diff
@@ -20,59 +21,50 @@ public static void main(String[] args) {
             System.out.print(" "+t+" ");
         }
         int y =12_000_000;
-        int people =12_000_000/1000;
         int born=17;
         int dead=8;
-
-        for(int t=1;t<=10;t++)
+        for( int year=1;year<=10;year++)
         {
-            y=y+(people*(born -dead));
-            System.out.println("год "+t+" численность населения состовляет "+y);
+            y=y+(y*(born -dead));
+            System.out.println("год "+year+" численность населения состовляет "+y);
         }
-        double total1 = 0;
-        double salary1 = 15000;
-        int d = 0;
-
-        for (; total1 < 12_000_000; d++) {
-            salary1 = salary1 * 1.07;
-            total1 = total1 * 1.07;
+        double total1=0;
+        double salary1=15000;
```

Comment:
```suggestion
        total=0;
        salary=15000;
```
новые переменные не нужны, надо просто обнулить уже ранее созданные